pkg/beholder/client.go

```diff
@@ -247,10 +247,17 @@ func newOtelResource(cfg Config) (resource *sdkresource.Resource, err error) {
 // Open question: what are pros/cons for using use map[]any vs use otellog.KeyValue
 func (e messageEmitter) Emit(ctx context.Context, body []byte, attrKVs ...any) error {
 	message := NewMessage(body, attrKVs...)
+	record := message.OtelRecord()
+
+	if _, ok := message.Attrs["timestamp"]; !ok {
```

Comment:
This feels like a separate concern than Emit - can we handle attribute hydration higher up the stack?